packages/carbon-graphs/src/js/controls/Bar/helpers/creationHelpers.js

```diff
@@ -82,47 +82,50 @@ const hasSecondaryDisplay = (dataTarget) => dataTarget.axisInfoRow.some(
  * @returns {object} dataTarget - Updated data target object
  */
 const setDataPoints = (graphConfig, dataTarget) => {
-  const { type } = graphConfig.axis.x;
+  const graphConfigTempParam = graphConfig;
+  const dataTargetTempParam = dataTarget;
```

Comment:
nit: Can we update the variable names to `graphConfigObject` instead of `graphConfigTempParam`? and also `dataTargetObject`? 
 
It is applicable to all places where we are using TempParam

Reply:
removed all temp params [here](https://github.com/cerner/terra-graphs/pull/34/commits/a853fdf7bfb9bf655bc7525ed3b46f2f1ef1bb64#diff-573c75d0840d9770792c112c53ea805dc9df8e03c6fea8f0a8117a23b8e9048fR76) and turned off the **no-param-reassign** linter rule.